### PR TITLE
feat: upgrade to semantic-release:v18.0.0

### PR DESCRIPTION
**Pull request by pritamstyz4ever**
## Context

All Screwdriver packages work with node:12 and semantic release needs to be upgraded to a latest version

## Objective

upgrade to semantic-release:v18.0.0

## References

https://github.com/semantic-release/semantic-release/releases/tag/v18.0.0

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: semantic_release_template.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 name: screwdriver-cd/semantic-release
 version: '1.0'
```
Comment (tkyi) [inline semantic_release_template.yaml]: not sure if should bump major version for template

File: semantic_release_template.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
-description: Template for semantic release
+description: Template for semantic release with Node 10 or higher
 maintainer: daolam112@gmail.com
```
Comment (tkyi) [inline semantic_release_template.yaml]: probably want to update this email

File: semantic_release_template.yaml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 config:
   image: node:6
```
Comment (tkyi) [inline semantic_release_template.yaml]: bump image?

**Review by tkyi — COMMENTED** [review]
some comments

File: semantic_release_template.yaml (hunk 2 above)
Comment (jithine) [reply semantic_release_template.yaml]: we could use `screwdriver-oss@yahooinc.com`

**Review by tkyi — APPROVED** [review]

**Pull request merged**